2 - RTB/Documentazione esterna/Verbali/23-12-15/23-12-15.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "15/12/23",
+  subTitle: "",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  externalParticipants : (
+    (name: "Alberton Michela", role: "Referente aziendale"),
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "Ruffin Filippo", role: "Referente aziendale"),
+  ), 
+  timeStart: "12:00",
+  timeEnd: "12:30",
+);
+
+= Ordine del giorno
+Ai referenti aziendali sono stati esposti gli avanzamenti svolti sui PoC rispetto al precedente meeting esterno e sono sorte discussioni relativamente a:
+- Analisi dei Requisiti;
+- tecnologie da utilizzare e utilizzate;
+- feature del PoC;
+- meeting futuri.
+
+== Considerazioni sull'Analisi dei Requisiti
+Non sono stati identificati particolari vincoli sulle performance.
+
+=== Vincoli sui browser 
+È richiesta la compatibilità dell'applicazione almeno con i browser:
+- Chrome (da versione 89);
+- Firefox;
+- Edge.
+
+Inoltre sono approvati ma non richiesti l'utilizzo di WebGL e la compatibilità dell'applicativo con i browser sopracitati versione mobile.
+
+=== Dispositivi utilizzati dall'utente  
+I dispositivi su cui sarà eseguita l'applicazione sono normali PC Desktop verosimilmente presenti in uffici ed eventualmente dispositivi tablet.
+
+== Considerazioni su tecnologie da utilizzare e utilizzate
+Durante il meeting sono state discusse ulteriormente le tecnologie impiegate e in programma per una futura implementazione, in particolare è stato approvato da parte del proponente l'utilizzo di:
+- Docker;
+- Next.js 
+- Express;
+- PostgreSQL.
+
+== Considerazioni su feature del PoC
+Relativamente ai PoC da realizzare in fututro e alla luce delle conoscenze acquisite, sono emersi con il proponente le seguenti considerazioni:
+
+- è emersa l'importanza di precaricare dal database una parte dei dati delle merci così da poter mostrare a schermo informazioni di base come quali sono i bin occupati ma le restanti informazioni delle singole merci necessiteranno di un'interrogazione al database on demand;
```

Comment:
dividerei la frase con un punto (o una virgola almeno) se no è troppo lunga

Reply:
In più si ripete 
``` md
sono emerse le seguenti considerazioni:
- è emersa ...
```